mix.exs

```diff
@@ -47,7 +47,7 @@ defmodule Vyasa.MixProject do
       {:postgrex, ">= 0.0.0"},
       {:phoenix_html, "~> 4.0"},
       {:phoenix_live_reload, "~> 1.5", only: :dev},
-      {:phoenix_live_view, "~> 0.20.17"},
+      {:phoenix_live_view, github: "phoenixframework/phoenix_live_view", ref: "440fd04", override: true},
```

Comment:
note to self: to support the stickyness behaviour across browser history navigation (i.e. pressing back / fwd button), we had to bump live_view version to the most updated. 


> [!WARNING]
> this means that managing the :live_admin dep ends up being manual